components/slider/index.css

```diff
@@ -674,14 +778,13 @@
 
 	.spectrum-Slider-handle {
 		border-color: var(--highcontrast-slider-handle-border-color-disabled, var(--mod-slider-handle-border-color-disabled, var(--spectrum-slider-handle-border-color-disabled)));
-		background: var(--highcontrast-slider-handle-disabled-background-color, var(--mod-slider-handle-disabled-background-color, var(--spectrum-slider-handle-disabled-background-color)));
+		background: var(--highcontrast-slider-handle-disabled-background-color, var(--mod-slider-handle-disabled-background-color, var(--spectrum-slider-handle-background-color-disabled)));
 		cursor: default;
 		pointer-events: none;
 
 		&:hover,
 		&:active {
-			/* @todo need to update --mod with a component-specific prefix */
-			border-color: var(--mod-disabled-border-color, var(--spectrum-disabled-border-color));
+			border-color: var(--mod-slider-disabled-border-color, var(--spectrum-disabled-border-color));
```

Comment:
Let's use the `--spectrum-slider-handle-border-color-disabled` variable you made here instead!